build-system/compile/post-closure-babel.js

```diff
@@ -80,26 +79,21 @@ function terserMinify(code) {
 }
 
 /**
- * Apply Babel Transforms on output from Closure Compuler, then cleanup added space with Terser.
+ * Apply Babel Transforms on output from Closure Compuler, then cleanup added
+ * space with Terser. Used only in esm mode.
  *
  * @param {string} directory directory this file lives in
- * @param {boolean} isEsmBuild
  * @return {!Promise}
  */
-exports.postClosureBabel = function (directory, isEsmBuild) {
-  const babelPlugins = conf.plugins({isPostCompile: true, isEsmBuild});
-
+exports.postClosureBabel = function (directory) {
   return through.obj(function (file, enc, next) {
-    if (path.extname(file.path) === '.map' || babelPlugins.length === 0) {
+    if (!argv.esm || path.extname(file.path) === '.map') {
```

Comment:
This is a nice quick change, but it does mean if we add a post closure change via babel that there are now two places to fix.

Don't think this is a problem, but worth calling out if we want to address in the future.

Reply:
Looks like we have no choice. Removing this guard yields the following error:
```
[13:29:29] TypeError: Cannot read property 'mappings' of null
    at decodeSourceMap (/Users/rsimha/src/amphtml/node_modules/@ampproject/remapping/dist/remapping.umd.js:178:28)
    at Array.map (<anonymous>)
    at buildSourceMapTree (/Users/rsimha/src/amphtml/node_modules/@ampproject/remapping/dist/remapping.umd.js:679:35)
    at remapping (/Users/rsimha/src/amphtml/node_modules/@ampproject/remapping/dist/remapping.umd.js:781:21)
    at DestroyableTransform._transform (/Users/rsimha/src/amphtml/build-system/compile/post-closure-babel.js:98:20)
    at DestroyableTransform.Transform._read (/Users/rsimha/src/amphtml/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:177:10)
    at DestroyableTransform.Transform._write (/Users/rsimha/src/amphtml/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:164:83)
    at doWrite (/Users/rsimha/src/amphtml/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:405:139)
    at writeOrBuffer (/Users/rsimha/src/amphtml/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:394:5)
    at DestroyableTransform.Writable.write (/Users/rsimha/src/amphtml/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:303:11)
    at Duplexify.ondata (/Users/rsimha/src/amphtml/node_modules/ternary-stream/node_modules/readable-stream/lib/_stream_readable.js:662:20)
    at Duplexify.emit (events.js:311:20)
    at Duplexify.EventEmitter.emit (domain.js:505:15)
    at addChunk (/Users/rsimha/src/amphtml/node_modules/ternary-stream/node_modules/readable-stream/lib/_stream_readable.js:297:12)
    at readableAddChunk (/Users/rsimha/src/amphtml/node_modules/ternary-stream/node_modules/readable-stream/lib/_stream_readable.js:279:11)
    at Duplexify.Readable.push (/Users/rsimha/src/amphtml/node_modules/ternary-stream/node_modules/readable-stream/lib/_stream_readable.js:240:10)
    at Duplexify._forward (/Users/rsimha/src/amphtml/node_modules/ternary-stream/node_modules/duplexify/index.js:172:26)
    at DestroyableTransform.onreadable (/Users/rsimha/src/amphtml/node_modules/ternary-stream/node_modules/duplexify/index.js:136:10)
    at DestroyableTransform.emit (events.js:311:20)
    at DestroyableTransform.EventEmitter.emit (domain.js:505:15)
    at emitReadable_ (/Users/rsimha/src/amphtml/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:520:12)
    at processTicksAndRejections (internal/process/task_queues.js:83:21)
```